mail_service/worker_notify_user.py

```diff
@@ -156,6 +160,25 @@ def notify_user_with_linter_results(job_info):
             job.delete()
             continue
 
+    if "build_failed" in job_info:
+        if job_info["build_failed"]:
```

Comment:
What is happening here? We're first checking if a key `build_failed` exists in `job_info` dictionary. And then we're checking if it's set to `True`? Can we just do the `True`/`False` check?